altair/expr/__init__.py

```diff
@@ -117,6 +117,115 @@ class expr(_ExprRef, metaclass=_ExprMeta):
+    .. _Math.atan:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/atan
+    .. _Math.atan2:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/atan2
+    .. _Math.ceil:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/ceil
+    .. _Math.cos:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/cos
+    .. _Math.exp:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/exp
+    .. _Math.floor:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/floor
+    .. _Math.hypot:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot
+    .. _Math.log:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/log
+    .. _Math.max:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max
+    .. _Math.min:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/min
+    .. _Math.pow:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow
+    .. _Math.random:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+    .. _Math.round:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round
+    .. _Math.sin:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sin
+    .. _Math.sqrt:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sqrt
+    .. _Math.tan:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/tan
+    .. _normal (Gaussian) probability distribution:
+       https://en.wikipedia.org/wiki/Normal_distribution
+    .. _cumulative distribution function:
+       https://en.wikipedia.org/wiki/Cumulative_distribution_function
+    .. _probability density function:
+       https://en.wikipedia.org/wiki/Probability_density_function
+    .. _log-normal probability distribution:
+       https://en.wikipedia.org/wiki/Log-normal_distribution
+    .. _continuous uniform probability distribution:
+       https://en.wikipedia.org/wiki/Uniform_distribution_(continuous
+    .. _*unit*:
+       ../api/time/#time-units
+    .. _JavaScript's String.replace:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace
+    .. _d3-format specifier:
+       https://github.com/d3/d3-format/
+    .. _*units*:
+       ../api/time/#time-units
+    .. _timeUnitSpecifier API documentation:
+       ../api/time/#timeUnitSpecifier
+    .. _timeFormat:
+       #timeFormat
+    .. _utcFormat:
+       #utcFormat
+    .. _d3-time-format specifier:
```

Comment:
@jonmmease these are going to be broken links, but there is code in `vega_expr.py` or `markup.py` that can fix them

Reply:
If you hook into this part you can piggyback off the same replacements

https://github.com/vega/altair/blob/b6481d2b580b8c3b90df835a26da35d73991fe08/tools/vega_expr.py#L252-L278

Reply:
Thanks, the `expand_urls` function is exactly what was needed. [47c578b](https://github.com/vega/altair/pull/3699/commits/47c578b668ab7f922ee09f5f603d0ef8f50f261d)